src/appdistribution/client.ts

```diff
@@ -54,11 +67,7 @@ export class AppDistributionClient {
 
   constructor(private readonly appId: string) {}
 
-  async getApp(
-    distributionFileType: DistributionFileType = DistributionFileType.APK
-  ): Promise<AppDistributionApp> {
-    utils.logBullet(`getting app details (Distribution type: ${distributionFileType})...`);
-    const appView = distributionFileType == DistributionFileType.AAB ? "FULL" : "BASIC";
+  async getApp(appView: AppView = AppView.BASIC): Promise<AppDistributionApp> {
```

Comment:
I _think_ the type here is redundant, but please feel free to ignore if it's required.

```suggestion
  async getApp(appView = AppView.BASIC): Promise<AppDistributionApp> {
```

Reply:
Nice! Updated